dataframely/schema.py

```diff
@@ -332,7 +336,7 @@ def sample(
                 validation_error = None
                 try:
                     cls.validate(relevant_rows)
-                except ValidationError as e:
+                except Exception as e:
```

Comment:
Can we further narrow this down? `ComputeError | ...`?

Reply:
Absolutely, this was surely a placeholder 😅